resource/ignite2019-prod/rules.json

```diff
@@ -22,11 +22,6 @@
           "ApplicableFrom": null,
           "ApplicableTo": null,
           "Identifier": "Session Catalog 2019"
-        },
-        {
-          "ApplicableFrom": null,
```

Comment:
Leave the 2018 entries to ensure that /videos will work

Reply:
@gavinbarron Will do. By the way, who else has access to approve these GitHub PRs?

Reply:
Unfortunately not at this stage.

If you get this in tonight I'll approve it before i leave for the second camping mission tomorrow morning.

Reply:
@gavinbarron Alright, I've updated this PR with the latest changes from the Orlando and MITT access rules excel.

If you could organise for someone to have access to Github to approve PR, then that'd be great too, as currently we're pointing to my forked repo for MITT to get Paris and Tokyo going (hubb changed some session type names and id in the session map)

Reply:
@gavinbarron Oh, sorry about this addendum, but apparently our v- accounts are sitting with you to approve. Could you spare a split sec to get those through? 
Cheers